dbt/include/snowflake/macros/utils/get_columns_spec_ddl.sql

```diff
@@ -0,0 +1,18 @@
+{% macro snowflake__get_columns_spec_ddl() %}
+  {# loop through user_provided_columns to create DDL with data types and constraints #}
+  {% if config.get('constraints_enabled', False) %}
+    {%- set ns = namespace(at_least_one_check=False) -%}
+    {%- set user_provided_columns = model['columns'] -%}
+    (
+    {% for i in user_provided_columns -%}
+      {%- set col = user_provided_columns[i] -%}
+      {% set constraints = col['constraints'] -%}
+      {%- set ns.at_least_one_check = ns.at_least_one_check or col['check'] %}
```

Comment:
```suggestion
      {%- set ns.at_least_one_check = ns.at_least_one_check or col['constraints_check'] %}
```
Do a find and replace on all check `configs` in all files to be `constraints_check` by name.